Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -1,7 +1,14 @@
 { Expression } = require './expression'
+{ FunctionRef } = require './reusable'
+{ decimalAdjust } = require '../util/math'
 { ValueSet, Code } = require '../datatypes/datatypes'
+{ Exception } = require '../datatypes/exception'
 { build } = require './builder'
+ucum = require  'ucum'
 
+module.exports.IncompatibleTypesException = IncompatibleTypesException = class IncompatibleTypesException extends Exception
+  constructor: (@a , @b , e) ->
+    super("Incompatible Types '" + @a + "' and '" + @b +"'", e)
```

Comment:
From Luke:

It looks like `@b + “’”, e` in CoffeeScript somehow gets turned into` this.b(+"'", e)` in JavaScript (very strange), which causes the error “this.b is not a function” to be thrown.
 
Changing the line to this seems to resolve the issue:
```super("Incompatible Types ‘#{@a}’ and ‘#{@b}’”, e)```